### ISSUE TITLE
saving and publishing options for other than Github like Netlify

### ISSUE THREAD

**Issue by sidmint** [silexlabs/Silex#1066]
I saw in a video that Guthub gives only one SSL cert/enforces https for only one domain name and not its sub domains but Netlify gives unlimited for sub domains too, is that true?

If so, then I/people may want to publish to Netlify in future and it'd be great if that's easy via built in saving and publishing options to do so for Netlify and the likes.

**Comment by lexoyo** [comment]
> I saw in a video that Guthub gives only one SSL cert/enforces https for only one domain name and not its sub domains but Netlify gives unlimited for sub domains too, is that true?

i don't think so... you can create several repos with a CNAME file pointing to different sub domains and github pages will generate a certificate for each site (see the repo settings)

> If so, then I/people may want to publish to Netlify in future and it'd be great if that's easy via built in saving and publishing options to do so for Netlify and the likes.

you already can : setup a new site in netlify and link it to the github repo where you publish. in silex, use the "select a folder" publication, not the "github pages" one

**Comment by sidmint** [comment]
oh wow, thank you.
